integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1302,6 +1306,148 @@ public Long fromDocumentIdentifier(String documentIdentifier,
 		}
 	}
 
+	@Test
+	public void customBridge_withParams_annotationMapping() {
+		class IndexedEmbeddedLevel1 {
+			@DocumentId(identifierBinder = @IdentifierBinderRef(type = ParametricBinder.class,
+					params = @Param(name = "stringBase", value = "3")))
+			Long theId;
+			@AssociationInverseSide(inversePath = @ObjectPath(@PropertyValue(propertyName = "level1")))
+			Object containing;
+		}
+		@Indexed(index = INDEX_NAME)
+		class IndexedEntity {
+			@DocumentId
+			Integer id;
+			@IndexedEmbedded(includeEmbeddedObjectId = true)
+			IndexedEmbeddedLevel1 level1;
+			public IndexedEntity(int id, Long level1Id) {
+				this.id = id;
+				this.level1 = new IndexedEmbeddedLevel1();
+				this.level1.theId = level1Id;
+				this.level1.containing = this;
+			}
+		}
+
+		backendMock.expectSchema( INDEX_NAME, b -> b
+				.objectField( "level1", b2 -> b2
+						// The ID is a long, and there is an identifier bridge on the DocumentId,
+						// so the ID field will be generated with the identifier bridge, and thus will be a String.
+						.field( "theId", String.class, b3 -> b3
+								.searchable( Searchable.YES ).projectable( Projectable.YES ) )
+				)
+		);
+		SearchMapping mapping = setupHelper.start().expectCustomBeans()
+				.setup( IndexedEntity.class, IndexedEmbeddedLevel1.class );
+		backendMock.verifyExpectationsMet();
+
+		doTestEmbeddedRuntime(
+				mapping,
+				id -> new IndexedEntity( id, 4242L ),
+				document -> document
+						.objectField( "level1", b2 -> b2
+								.field( "theId", "4245" ) )
+		);
+	}
+
+	@Test
+	public void customBridge_withParams_programmaticMapping() {
```

Comment:
For consistency with the test methods above...

```suggestion
	public void includeEmbeddedObjectId_identifierBridge_withParams_programmaticMapping() {
```